I'm sorry, but an uncaught exception occurred.

While running game code:
Exception: Style 'main_menu_button_large2' does not exist.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in script
    python hide:
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\ast.py", line 1138, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\python.py", line 1122, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/_layout/screen_main_menu.rpym", line 28, in <module>
    python hide:
  File "renpy/common/_layout/screen_main_menu.rpym", line 35, in _execute_python_hide
    ui.interact()
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 3582, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, pause_modal=pause_modal, **kwargs) # type: ignore
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 4055, in interact_core
    root_widget.visit_all(lambda d : d.per_interact())
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\screen.py", line 481, in visit_all
    self.child.visit_all(callback, seen=None)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 684, in visit_all
    d.visit_all(callback, seen)
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\core.py", line 674, in visit_all
    for d in self.visit():
  File "D:\all_Kvasych_games\cren'pie\renpy-8.1.3-sdk\renpy\display\layout.py", line 1258, in visit
    self.style._visit_window(rv.append)
  File "style.pyx", line 541, in renpy.style.StyleCore._visit_window
  File "style.pyx", line 528, in renpy.style.StyleCore._get_unoffset
  File "style.pyx", line 493, in renpy.style.StyleCore._get
  File "style.pyx", line 679, in renpy.style.build_style
  File "style.pyx", line 142, in renpy.style.get_full_style
  File "style.pyx", line 98, in renpy.style.get_style
Exception: Style 'main_menu_button_large2' does not exist.

Windows-10-10.0.19044 AMD64
Ren'Py 8.1.3.23091805
Сохры 1.0
Wed Jan 15 17:41:30 2025
